spec/spec_helper.rb

```diff
@@ -1,10 +1,16 @@
 ENV['RAILS_ENV'] = 'test'
 
-require File.expand_path('../dummy/config/environment.rb',  __FILE__)
-
 require 'pry'
+require 'active_model/railtie'
+require 'action_controller/railtie'
+require 'action_view/railtie'
+require 'actionpack/action_caching/railtie'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.